.github/CODEOWNERS

```diff
@@ -1,16 +1,13 @@
 # Automatically generated CODEOWNERS
 # Regenerate with `make update-codeowners`
-draft-ietf-httpbis-bcp56bis.md mnot@mnot.net
-draft-ietf-httpbis-cache-header.md mnot@mnot.net
+ mt@lowentropy.net caw@heapingbits.net
```

Comment:
this doesn't seem correct because there is no draft. Not sure MT or Chris will welcome becoming codeowner for everything

Reply:
😱

Reply:
FWIW this commit was made via `http-extensions$ make update-codeowners`. @martinthomson or @mnot might want to figure out what caused the script to bug out

Reply:
At a meeting, but I'll take a look at it when I am able.